src/Umbraco.Tests.UnitTests/Umbraco.Core/ShortStringHelper/StringExtensionsTests.cs

```diff
@@ -293,14 +293,25 @@ public void ReplaceManyWithCharMap()
             Assert.AreEqual(expected, output);
         }
 
-        #region Cases
         [TestCase("val$id!ate|this|str'ing", "$!'", '-', "val-id-ate|this|str-ing")]
         [TestCase("val$id!ate|this|str'ing", "$!'", '*', "val*id*ate|this|str*ing")]
-        #endregion
         public void ReplaceManyByOneChar(string input, string toReplace, char replacement, string expected)
         {
             var output = input.ReplaceMany(toReplace.ToArray(), replacement);
             Assert.AreEqual(expected, output);
         }
+
+        [TestCase("", false)]
+        [TestCase("1234", false)]
+        [TestCase("6:18", true)]
+        [TestCase("12:34", true)]
+        [TestCase("25:03", false)]
+        [TestCase("18:61", false)]
+        [TestCase("1:14:23", false)]
```

Comment:
`TimeSpan.Parse("1:14:23")` is valid. 1 hr, 14 min, 23 sec

Reply:
Have reverted my change, and made it clear this is validating a `TimsSpan`, not just a time.